app/src/main/java/com/android/sample/ui/authentication/AvailabilityScreen.kt

```diff
@@ -0,0 +1,174 @@
+package com.android.sample.ui.authentication
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.filled.ArrowBack
+import androidx.compose.material3.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.android.sample.model.profile.AcademicLevel
+import com.android.sample.model.profile.Profile
+import com.android.sample.model.profile.Role
+import com.android.sample.model.profile.Section
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun AvailabilityScreen(profile: Profile, onProfileUpdate: (Profile) -> Unit) {
```

Comment:
maybe you forget to push the corresponding Profile data class since it does not exist already in the main branch.

Reply:
Yes thank you, I've added Lea's file now it should work!